data/src/main/java/me/androidbox/data/local/converter/EventConverter.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.converter
+
+import androidx.room.TypeConverter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import me.androidbox.data.remote.model.request.EventRequestDto
+import javax.inject.Inject
+
+class EventConverter @Inject constructor(){
+    private val moshi = Moshi.Builder()
+        .add(KotlinJsonAdapterFactory())
+        .build()
+
+    private val jsonAdapter = moshi.adapter(EventRequestDto::class.java)
+
+    @TypeConverter
+    fun fromJson(json: String): EventRequestDto? {
+        return jsonAdapter.fromJson(json)
+    }
+
+    @TypeConverter
+    fun toJson(eventRequestDto: EventRequestDto): String {
+        return jsonAdapter.toJson(eventRequestDto)
+    }
+}
```

Comment:
TypeConverters are only used for Room. You don't need this to manually serialize an object for a network request

Reply:
@philipplackner So I should be doing something like the following:
```
        /* Serialize the event request to json */
        val moshi = Moshi.Builder().build()
        val jsonAdapter = moshi.adapter<EventRequestDto>(EventRequestDto::class.java)
        val eventRequestJson = jsonAdapter.toJson(eventRequest)
```

Reply:
Yes 👍